tests/test_tracking_lattice/test_tracking_lattice.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+
+import os
+import sys
+import math
+from collections import OrderedDict
+sys.path.insert(0, os.pardir)
+sys.path.insert(0, os.path.join(os.pardir, 'openmoc'))
+from testing_harness import TrackingTestHarness
+from input_set import LatticeGridInput
+
+import openmoc
+
+class TrackingLatticeGridTestHarness(TrackingTestHarness):
```

Comment:
We already have ``SimpleLattice`` so ``LatticeGrid`` implies that it is a ``Lattice`` but it implicitly forms a grid geometry (the same geometry as ``Grid``)